test/environment.rb

```diff
@@ -118,6 +119,23 @@ def self.create_test_nodes model, depth, width, parent = nil
       end
     else; []; end
   end
+
+  private
+
+  def self.use_optimizer
+    use_optimizer = ENV['OPTIMIZER'] == 'true'
+    return unless use_optimizer
+
+    if db_type == 'pg'
+      require File.expand_path('../../lib/ancestry/optimizers/pg', __FILE__)
```

Comment:
I think this can be required regardless of the database type.

We just won't include it unless we want the optimization